base/float.jl

```diff
@@ -713,6 +713,8 @@ end
     eps(::Type{Float16}) = $(bitcast(Float16, 0x1400))
     eps(::Type{Float32}) = $(bitcast(Float32, 0x34000000))
     eps(::Type{Float64}) = $(bitcast(Float64, 0x3cb0000000000000))
+    eps(::Type{Complex{T}}) where {T<:AbstractFloat} = eps(T)
+    eps(x::Complex{T}) where {T<:AbstractFloat} = eps(norm(x))
```

Comment:
Why not go with the correct definition of `hypot(eps(real(z)), eps(imag(z)))` rather than the approximate value of `eps(norm(z))` and document this property as the defining characteristic for real and complex number types? IIRC, @simonbyrne already put some effort into documenting this very property for real values, so extending it to complexes makes sense.